File: sm.py

from pubnub.pubnub import PubNub
from pubnub.pnconfiguration import PNConfiguration
from pubnub.callbacks import SubscribeCallback
from pubnub.enums import PNStatusCategory

import uuid
 






pnconfig = PNConfiguration()
pnconfig.subscribe_key = "sub-c-5bdd008c-6d97-11e8-a49b-66b3abd5adf6"
pnconfig.publish_key = "pub-c-2bda6cf3-c455-4b28-bbed-00611b461f98"
pnconfig.ssl = False
 
pubnub = PubNub(pnconfig)




#publish
def publish_callback(result, status):
    pass
    # Handle PNPublishResult and PNStatus
    print(result)
    print(status)
 
pubnub.publish().channel('ch1').message(['hello', 'there']).async(publish_callback)

File: sm.js
 //  var five = require ('johnny-five');
 // var board = new five.Board();

var PubNub = require('pubnub')




    //board.on("ready", function(){
 //  var led_green = new five.Led(13);
  // var led_red = new five.Led(3);
 pubnub = new PubNub({
		
		publishKey : 'pub-c-2bda6cf3-c455-4b28-bbed-00611b461f98',
        subscribeKey : 'sub-c-5bdd008c-6d97-11e8-a49b-66b3abd5adf6',
               
                    });


               publish("hel;kae");
                    pubnub.addListener(
                    {
                        
                         message: function(message) 
                         {
                            console.log(message.message);
                            

                              }   
                         
                         
                    });    
                 
    console.log("Subscribing");
    pubnub.subscribe({
        channels: ['ch2'] 
       

    });

   
   
function publish(x){
             

                     var publishConfig = 
                     {
                        channel : "ch1",
                        message : {
                                        
                                        "text": "d"
                                   }
                     }
                     pubnub.publish(publishConfig, function(status, response) 
                     {
                     console.log(status, response);

                     });

                 
                        };
   
//});
